angularjs-autogrow.js

```diff
@@ -62,6 +62,30 @@
 
                 $scope.$watch($attrs.ngModel, $scope.autogrowFn);
 
+                // Extract css properties to spy on
+                var spyProps = $attrs.autogrow.split(',');
+                angular.forEach(spyProps, function(property) {
+                  // Set a watcher on each property
+                  $scope.$watch(
+                    function() {
+                      return $element.css(property);
+                    },
+                    styleChangedCallBack,
+                    true
```

Comment:
Just checked angular doc and it's not needed for this use case. I'm going to update the PR